src/Commands/ExportPostmanCommand.php

```diff
@@ -211,14 +211,33 @@ protected function buildTree(array &$routes, array $segments, array $request): v
 
     public function makeRequest($route, $method, $routeHeaders, $requestRules)
     {
+        preg_match_all('/(?={([[:alnum:]]+)})/si', $route->uri(), $variables, PREG_SET_ORDER, 0);
+
+        if (count($variables)) {
+            // Flatten Matches and Filter Empty Values
+            $variables = array_filter(array_merge(...array_values($variables)));
+        }
```

Comment:
Couldn't quite figure out how to fully exclude the outer group with the regular expression. If we can do that, we could remove `array_filter` from the above.

![image](https://user-images.githubusercontent.com/238022/146281956-58960777-8a1f-4354-a541-240d78075f14.png)